docs/pages/per_address_priority.md

```diff
@@ -0,0 +1,54 @@
+11.	Setting a priority to zero may result in source loss behavior for the corresponding address if there is no other source that gets control.  
+It may be desirable for sources to provide a user configurable option to also set the data to 0 before setting the priority to 0.
+12. For some compatibility with receivers that don’t support the 0xDD packets, sources sending 0xDD packets shall also provide a packet priority value that applies to all addresses.  The default value for this priority shall be 100.  This allows for packet priority and HTP control between sources.
+13. When a receiver detects a new source, it waits for a 0xDD packet for up to 1.5 seconds before processing DMX levels from that source.  If a receiver does not detect a 0xDD packet, it fails back and uses the packet priority until a 0xDD packet is detected.   
+14.	When a receiver detects that a source has reached the sACN source loss timeout for 0xDD packets (e.g. no 0xDD packets in Universal Hold Last Look Time--2.5 seconds at minimum), but is still receiving NULL start code packets, the receiver shall fall back to using the sACN packet priority located in the NULL start code packets.
+15. When a receiver detects that a source has reached the sACN source loss timeout for NULL start code packets (e.g. no NULL start code packets in the Universal Hold Last Look Time--2.5 seconds at minimum) the receiver shall act as if the source has given up control of its addresses as stated in 4.16 even if 0xDD packets continue to be received.  
+The desired sequence for a source to terminate its sACN stream is to send three packets with the Stream_Terminated bit (6) of the Packet options field set to 1.  This will allow the device with the next highest priority to gain control after the appropriate sampling time resolution.
+16. When a receiver detects that a source has given up control of an address (by setting the priority for that address to 0), the receiver will give control to the source with next highest priority. If no source is available, source loss behavior for that address shall apply.
+
+# Summary
+In summary, priority may be specified on a per packet basis in sACN. This ETC extension allows priorities to be provided on a per-address basis using the alternate start code of 0xDD.
+
+With this extension, taking control is accomplished by setting a non-zero priority for an address.  Giving up control is the reverse, setting the priority to 0.
+
+Overall, receivers wait for a short period on start up to see if the alternate start code priority information is being provided before they start accepting data.
+
+If no alternate start code priority information arrives, they use the packet priority.  If the packet is a Draft sACN packet and it specifies that it does not carry a priority, by placing a zero in the field.
```

Comment:
I don't think we need to reference draft any more.

Reply:
Agreed